tests/components/nightscout/test_config_flow.py

```diff
@@ -0,0 +1,108 @@
+"""Test the Nightscout config flow."""
+from aiohttp import ClientConnectionError
+
+from homeassistant import config_entries, data_entry_flow, setup
+from homeassistant.components.nightscout.const import DOMAIN
+from homeassistant.components.nightscout.utils import hash_from_url
+from homeassistant.const import CONF_URL
+
+from tests.async_mock import patch
+from tests.common import MockConfigEntry
+from tests.components.nightscout import GLUCOSE_READINGS, SERVER_STATUS
+
+CONFIG = {CONF_URL: "https://some.url:1234"}
+
+
+async def test_form(hass):
+    """Test we get the user initiated form."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5985897)